prysm/x/optym/activation.py

```diff
@@ -206,3 +206,110 @@ def discretize(self, x):
         # take argmax along dim k, and take that from levels
         indices = np.argmax(encoded, axis=-1)
         return np.take(self.levels, indices)
+    
+
+class Tanh:
+    """Tanh activation function.
+
+    Uses the hyperbolic tangent function f(x) = tanh(x). Constructed to support additional
```

Comment:
For all of these functions, the docstring is of the form

```
"""Name to satisfy linter.

Uses the f(x) = the_f(x)
...
"""
```

I think the line starting with Uses is not value added?

I would do just

```
class Tanh:
"""Tanh(x).

def __init__(self, a=1, x0=0, y0=0): pass
```